src/react/endpoint/EndpointList.tsx

```diff
@@ -64,8 +64,13 @@ function EndpointList({ endpoints, locations }: Props) {
         },
         Cell({ value: hostName }: { value: Hostname }) {
           return (
-            <div style={{ display: 'flex' }}>
-              <span style={{ paddingRight: spacing.sp14 }}>{hostName}</span>
+            <div style={{ display: 'flex', width: `calc(100% - 100px)` }}>
```

Comment:
Why `-100px` ?

Reply:
to align icons regardless of text length

![Screenshot 2023-07-10 at 17 50 47](https://github.com/scality/zenko-ui/assets/135591453/eedebd4c-2e12-4637-891e-fde23fd22ea6)

Reply:
Can't we replace it by a `<Wrap />` ?